pkg/sriov/poolconfig.go

```diff
@@ -255,6 +255,29 @@ func (builder *PoolConfigBuilder) WithMaxUnavailable(maxUnavailable intstrutil.I
 	return builder
 }
 
+// WithRdmaMode method sets rdmaMode to shared/exclusive.
+func (builder *PoolConfigBuilder) WithRdmaMode(mode string) *PoolConfigBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	if mode == "" {
+		builder.errorMsg = "RdmaMode cannot be empty"
+
+		return builder
+	}
+
+	if mode != "shared" && mode != "exclusive" {
+		builder.errorMsg = "Invalid value for rdmaMode. It should be 'shared' or 'exclusive'"
```

Comment:
and for both of the error messages could you please change them to start with a lowercase letter?
```suggestion
		builder.errorMsg = "invalid value for rdmaMode. It should be 'shared' or 'exclusive'"
```